content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Nota:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> Los `Web Component` a menudo exponen una API imperativa. Por ejemplo, un `Web Component` `video` podría exponer las funciones `play()` y `pause()`. Para acceder a un API imperativa de un `Web Component`, necesitarás utilizar una referencia para interactuar con el DOM directamente. Si estás utilizando `Web Component` de terceros, lo mejor sería escribir un componente React que sirva como un contenedor para tu `Web Component`.
 >
-> Events emitted by a Web Component may not properly propagate through a React render tree.
-> You will need to manually attach event handlers to handle these events within your React components.
+> Los eventos emitidos por un `Web Component` pueden no distribuirse correctamente a través de un árbol de renderizado React.
+> Deberás agregar manualmente los controladores de eventos para manejarlos dentro de tus componentes React.
 
-One common confusion is that Web Components use "class" instead of "className".
+Una cosa que puede confundirte es que los `Web Component` usan ***"class"*** en vez de ***"className"***
```

Comment:
Una cosa que puede confundirte, es que los `Web Components` usan... I think there is comma missing after `confundirte` and plural again for `Web Components`